src/AccessibilityInsights.Extensions.GitHubUnitTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("AccessibilityInsights.Extensions.GitHubUnitTests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("HP Inc.")]
```

Comment:
I'm not sure we want to hand over our copyright to HP. Also below.

Reply:
Fixed, forgot to change it, whenever I create a new project, the PC manufacture is added as company by default!